src/Api/HealthChecks/HealthCheckResponseWriter.cs

```diff
@@ -0,0 +1,48 @@
+using System.Text;
+using System.Text.Json;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+
+namespace Passwordless.Api.HealthChecks;
+
+public static class HealthCheckResponseWriter
+{
+    public static Task WriteResponse(HttpContext context, HealthReport healthReport)
```

Comment:
Do we follow this convention?

```suggestion
    public static Task WriteResponseAsync(HttpContext context, HealthReport healthReport)
```

Reply:
Just copy pasted the sample from Microsoft.